lib/widgets/conversations/add_conversation_widget.dart

```diff
@@ -43,7 +37,8 @@ class AddConversationWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     if (conversation != null) {
       textController.text = conversation.name;
-      textController.selection = TextSelection(baseOffset: 0, extentOffset: conversation.name.length);
+      textController.selection =
+          TextSelection(baseOffset: 0, extentOffset: conversation.name.length);
```

Comment:
again, is this due to a linter?

Reply:
Same as the last one, there should actually be a trailing comma after conversation.name.length to cause the object to wrap rather than the line, I'll update now